cypress/integration/ui/sign.test.ts

```diff
@@ -2,20 +2,30 @@
 /// <reference types="@testing-library/cypress"/>
 import { paymentXdr, sourceAccount, paymentRecipient } from '../../fixtures/sign.json';
 
-const url = Cypress.env('HOST');
 describe('checks that the /sign component works', () => {
-    it('alerts you if you do not pass an XDR as a query parameter or it is invalid', () => {
-        cy.visit(url);
-        cy.get('a[href*="sign"]').click();
-        cy.findByText('INVALID OR NULL XDR').should('exist');
+    const url = Cypress.env('HOST');
+    const testSecretKey = Cypress.env('PRIVATE');
```

Comment:
I think this should be something more indicative like TEST_PRIVATE_KEY or something like that